lib/cppcheck.cpp

```diff
@@ -1033,22 +1047,28 @@ unsigned int CppCheck::checkFile(const std::string& filename, const std::string
     return mExitCode;
 }
 
-// TODO: replace with ErrorMessage::fromInternalError()
-void CppCheck::internalError(const std::string &filename, const std::string &msg)
+static ErrorMessage makeError(const std::string &filename, const std::string &msg, const std::string &id)
 {
-    const std::string fullmsg("Bailing out from analysis: " + msg);
-
     const ErrorMessage::FileLocation loc1(filename, 0, 0);
     std::list<ErrorMessage::FileLocation> callstack(1, loc1);
 
-    ErrorMessage errmsg(callstack,
+    return ErrorMessage(callstack,
                         emptyString,
                         Severity::error,
-                        fullmsg,
-                        "internalError",
+                        msg,
+                        id,
                         Certainty::normal);
+}
+
+// TODO: replace with ErrorMessage::fromInternalError()
+void CppCheck::internalError(const std::string &filename, const std::string &msg)
+{
+    mErrorLogger.reportErr(makeError(filename, msg, "internalError"));
```

Comment:
I suggest you add the "Bailing out from analysis:" here:

     mErrorLogger.reportErr(makeError(filename, "Bailing out from analysis:" + msg, "internalError"));